apps/frontend/app/(client)/(main)/course/[courseId]/_components/RecentNotice.tsx

```diff
@@ -82,18 +108,25 @@ export function RecentNotice() {
               //TODO: 공지 요소 누르면 해당 페이지로 넘어가는 기능 필요!
               <li
                 key={notice.id}
-                className="flex border-b pb-1 text-sm text-gray-700"
+                className="flex justify-between border-b text-sm"
               >
-                <span
-                  className={`mr-2 text-xs ${
-                    notice.isNew ? 'text-green-500' : 'text-red-500'
-                  }`}
-                >
-                  ●
-                </span>
-                <span className={notice.isNew ? 'font-bold' : ''}>
-                  {notice.title}
-                </span>
+                <div>
+                  <span
+                    className={`mr-2 text-xs ${
+                      notice.isNew ? 'text-primary' : 'text-[#8A8A8A]'
+                    }`}
+                  >
```

Comment:
cn 함수를 사용하면 가독성을 높일 수 있어요!
```suggestion
                  <span
                    className={cn(
                      'mr-2 text-xs',
                      notice.isNew ? 'text-primary' : 'text-[#8A8A8A]'
                    }}
                  >
```